bnb/recherche.php

```diff
@@ -214,7 +214,7 @@
             $mapDate = array(3242 => "08/05/2013", 3241 => "01/05/2013", 3240 => "24/04/2013", 3239 => "17/04/2013", 3238 => "10/04/2013", 3237 => "03/04/2013", 3236 => "20/03/2013", 3231 => "13/02/2013");
 
             $files = glob('BNBDaten/*.rdf');
-            usort($files, function ($a, $b) {
+            usort($files, function($a, $b) {
```

Comment:
Travis complains here:
```
----------------------------------------------------------------------

 217 | ERROR | [x] Expected 1 space after FUNCTION keyword; 0 found

     |       |     (Squiz.Functions.MultiLineFunctionDeclaration.SpaceAfterFunction)

----------------------------------------------------------------------

PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY

----------------------------------------------------------------------
```

Should we exclude that rule in https://github.com/UB-Mannheim/malibu/blob/master/dist/ruleset.xml ?

Reply:
Hm... I think we should instead activate the spaces before closure parameter in Scrutinizer, because this is an anonymous function here.